### PR TITLE
chore(Python): added is unique character algorithm

### PR DESCRIPTION
**Pull request by Dhee2211**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

Added  algorithm to check whether the given string has unique characters or not

Algorithm is written in python language. 

**Other information:**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,5 @@ Thumbs.db
 *.class
 *.idea
 
+
+algorithms/Python/arrays/unique_character.py
```
Comment (Arsenic-ATG) [inline .gitignore:16]: looks like you accidentally created this project somewhere else and didn't wanted git to track that particular file. 

it's recommended for `.gitignore` to only contain files which are unnecessary and could be generated when working with the project itself (for example: compiler generated files, project files etc. ). 

If you want some files to be untracked exclusively on your local git and don't want to pollute `.gitignore` then it's recommended to update `.git/info/exclude` instead.

**Review by Arsenic-ATG — CHANGES_REQUESTED** [review]
The program works correctly.

Just remove the unnecessary content from `.gitignore` and update the [index README.md file](https://github.com/Dhee2211/DSA/blob/dhee2211-linked-list-examples/algorithms/Python/README.md) with your creation to make this eligible for the merging.

File: .gitignore (hunk 1 above)
Comment (Dhee2211) [reply .gitignore:16]: I will update shortly. Thank you for the review.

**Review by Dhee2211 — COMMENTED** [review]
Updated  .gitignore and README.md

**Review by Arsenic-ATG — APPROVED** [review]

**Pull request merged**